content/en/docs/installation/upgrading/upgrading-0.15-0.16.md

```diff
@@ -5,5 +5,26 @@ weight: 840
 type: "docs"
 ---
 
-When upgrading from `v0.15` to `v0.16`, no special upgrade steps are required 🎉.
+## Issue with older versions of `kubectl`
+`kubectl` versions lower than `v1.19.0-rc.1` have issues updating the `v0.16` CRD files, this is due to [a bug when handling deeply nested CRDs](https://github.com/kubernetes/kubernetes/issues/91615).
+This patch is being backported into older versions of `kubectl` as new patch releases soon, at time of writing this was not the case.
+This bug will make `kubectl apply -f [...]` hang. 
+
+This bug only happens if you try to re-apply the `v0.16` CRDs, initial upgrade will not cause any issues. If you have this issue please use the `v1.19.0-rc.2` version of `kubectl` to apply this.
```

Comment:
This bug only happens during a re-apply of the `v0.16` CRDs. Initial upgrade does not cause issues.